components/cronet/cronet_global_state_stubs.cc

```diff
@@ -63,9 +64,16 @@ void PostTaskToInitThread(const base::Location& posted_from,
 }
 
 std::unique_ptr<net::ProxyConfigService> CreateProxyConfigService(
+    const std::string proxy_server,
     const scoped_refptr<base::SequencedTaskRunner>& io_task_runner) {
-  return net::ProxyConfigService::CreateSystemProxyConfigService(
-      io_task_runner);
+  if (proxy_server.empty()) {
+      return net::ProxyConfigService::CreateSystemProxyConfigService(io_task_runner);
+  } else {
+      net::ProxyConfig proxy_config;
+      proxy_config.proxy_rules().ParseFromString(proxy_server);
```

Comment:
From the description of this class
[L18](https://github.com/weblifeio/chromium/pull/1/files#diff-97e56188efb786ed3ba7cb71312c5bd19aff5d21b3e08ca9c923b38f4c5b5bf2L18)
it seems this stub is majorly used for build with tests. Did you check already that this does get executed fine and proxy gets added to URL Request Context for proper request handling?

Reply:
yes, I  checked from all sides, the fixed proxy is used in Cronet when configured. Also all unit tests pass.

Reply:
Ohh that's great :)